### PR TITLE
Add prelude guidelines via commenter action

### PR DESCRIPTION
**Pull request by mjgpy3**
Example comment body as markdown: https://gist.github.com/mjgpy3/94b3c1db858f968dbc17dad12770959f

### CODE REVIEW

**Review by eborden — APPROVED** [review]
Thanks for putting this together. This is a great starting point for defining rules of the road for our `Prelude`.

File: .github/commenter.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Layer-specific super-commonly-used imports should be kept in a prelude
+    specific to that layer.
+
+    Library-specific super-commonly-used imports should, ideally, be (1)
+    upstreamed, (2) put in their own package somewhere or, if need be, (3)
+    housed somewhere near their usage. You may want to consider a `freckle-app`
+    module other than the prelude as a potential home for such imports.
+
+      [func-on]: https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Function.html#v:on
+      [esq-on]: https://hackage.haskell.org/package/esqueleto-3.5.3.0/docs/Database-Esqueleto.html#v:on
```
Comment (pbrisbin) [inline .github/commenter.yml:61]: This looks a little over-indented:

```suggestion
    [func-on]: https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Function.html#v:on
    [esq-on]: https://hackage.haskell.org/package/esqueleto-3.5.3.0/docs/Database-Esqueleto.html#v:on
```

File: .github/commenter.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    **Idiomatic**
+
+    We'd like to keep the codebase accessible to Frecklers and, more broadly,
+    Haskellers. We're open to trying new things but, until they've been proven
+    out a bit, consistency is key!
+
+    This concept can be a little squishy but try to think along the lines of
+
+    > If I learned Haskell by contributing to the Freckle codebase, am I likely
+      to be comfortable with the pattern(s) this export encourages?
```
Comment (pbrisbin) [inline .github/commenter.yml:40]: If you're hard-breaking generally (which I like), I think this should be like this:
```suggestion
    > to be comfortable with the pattern(s) this export encourages?
```

And ditto L45.

**Review by StevenXL — APPROVED** [review]
💪

**Review by pbrisbin — APPROVED** [review]
Super cool!

**Pull request merged**